### PR TITLE
Newest upgrade on offline mode

### PR DESCRIPTION
**Pull request by SRPBerenyiM**

### CODE REVIEW

File: game.exe
Comment (joshika39) [inline game.exe:1]: Remove the file!

File: game.o
Comment (joshika39) [inline game.o:1]: Remove the file!

**Review by joshika39 — CHANGES_REQUESTED** [review]
Add the `game.exe` and `game.o` to the .gitignore file

File: game.exe
Comment (joshika39) [reply game.exe:1]: By Removing I mean, delete the .exe and push it that way!

File: game.o
Comment (joshika39) [reply game.o:1]: By Removing I mean, delete the .o and push it that way!

**Review by joshika39 — CHANGES_REQUESTED** [review]

File: game.o
Comment (SRPBerenyiM) [reply game.o:1]: I forgot how to do that.

Comment (joshika39) [reply game.o:1]: If you go to the project folder you will see a `game.cpp`, a `game.exe` and a `game.o`. Now, delete the `.exe` and the `.o` files. 

After you've deleted the files, go to Fork, and do a commit and a push as usual.

Comment (joshika39) [reply game.o:1]: Those two, yes.

**Review by joshika39 — APPROVED** [review]

**Pull request merged**